### DISCUSSION TITLE
请问是否有可能将原项目的一些插件安装到本项目上呢？

### DISCUSSION THREAD

**Discussion by CeciliaAki** [Arondight/Adachi-BOT#797] [Q&A] [answered]
如题。

我不知道 1.这是否被允许
                2.如果被允许应当怎样操作

我对原项目当中的聊天插件非常感兴趣，但使用了git clone以后对API的设置有些头疼，希望能够获得各位大佬的指点，如果这样的操作是不被允许的，我也会停止操作。

**Comment by mark9804** [comment, marked as answer]
架构不一样，你可以照猫画虎仿写一个

**Reply by CeciliaAki** [reply]
咳咳，这对目前的我来说还是太困难了，不过我会尝试去研究一下，谢谢您指点方向。

**Reply by Arondight** [reply]
照猫画虎可以参考项目的开发文档，原项目架构比较复杂，数据上有一个前后端无状态交互，数据由 bot 管理并在网页里按需多次载入。这个项目没有数据交互，所有的数据都是单向通过 url 一次性灌到网页，换言之以浏览器为界限来看 bot 和网页之间没一个统一的数据源。加上门槛比较低的 es6 语法，二次开发应该会简单很多。

**Reply by Arondight** [reply]
另外原项目当前的架构是两级插件，原神插件去管理自己的插件，这个项目是一级插件，功能层级上是扁平的，这个也需要注意一下，做二次开发直接按照手册写插件即可，不需要和原项目一样做二级插件，架构不太好，不过逻辑上明了一些

**Reply by CeciliaAki** [reply]
收到，谢谢两位大佬的提醒！

**Comment by Arondight** [comment]
不可能，用的语言都不一样。
